deploy/helm/events-collector-config.yaml

```diff
@@ -324,14 +329,16 @@ service:
         - resource/manifest
         - swk8sattributes
         - transform/cleanup_attributes_for_nonexisting_entities
+{{- if not (and .Values.otel.events.enabled .Values.otel.manifests.enabled) }}
```

Comment:
This means that in the case of disabled event/manifest collection, we will still collects and process everything, just throw almost everything out at the end. It will work, just not efficiently.

Reply:
I will create a follow up PR for this